scripts/bootstrap/buildenv.sh

```diff
@@ -301,3 +301,15 @@ function get_bind_target() {
   $BAZEL --bazelrc=${BAZELRC} --nomaster_bazelrc ${BAZEL_DIR_STARTUP_OPTIONS} \
     query "deps($1, 1) - $1"
 }
+
+# Create a link for a directory on the filesystem
+function link_dir() {
+  local source=$1
+  local dest=$2
+
+  if [ "${PLATFORM}" == "windows" ]; then
+    cmd.exe /C 'mklink /J "${dest}" "${source}"'
```

Comment:
Sorry I gave the snippet wrong. This should be:

    cmd.exe /C "mklink /J \"$(cygpath "$dest")\" \"$(cygpath "$source")\""

Explanation:
- need to double-quote the whole `mklink` expression to pass it as one argument but allow Bash variable expansion
- need to double quote `mklink`'s arguments (cmd.exe doesn't support single quotes), but we need to Bash-escape these double quotes
- need to use cygpath to pass a Windows path to `mklink`

Reply:
Thanks for the help, pushed this change.